FCCee/IDEA/compact/IDEA_o1_v03/DectDimensions_IDEA_o1_v03.xml

```diff
@@ -214,6 +178,15 @@
 
     <constant name="tracker_region_zmax" value="DCH_half_length_total"/>
     <constant name="tracker_region_rmax" value="DCH_outer_cyl_R_total"/>
+    
+    <!-- Fiber dual-readout calorimeter tower dim-->
+    <constant name="FiberDRCalo_barrel_height"       value="2.*m"/>
+    <constant name="FiberDRCalo_barrel_inner_radius" value="2.5*m"/>
+    <constant name="FiberDRCalo_barrel_nphi"         value="144"/>
+    <constant name="FiberDRCalo_endcap_height"       value="2.*m"/>
+    <constant name="FiberDRCalo_endcap_inner_radius" value="2.5*m"/>
```

Comment:
Would you mind changing `FiberDRCalo_endcap_inner_radius` to `FiberDRCalo_endcap_z_start`? (as discussed in the mattermost)